tests/functional/test_zipped_botocore.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We need to make sure we're using accurate copyright dates so we're not misattributing new code to the past. Overall though, we should probably keep this out of the normal test suite. Let's try to move these checks to `run-zip-tests` to establish state before we run checks.

Reply:
yep i've removed this file and move some state checks to `run-zip-tests`. I've removed any attempts to load data or service models in there as the unit tests will handle those checks after establishing a zipped state.